basics/crud.rb

```diff
@@ -0,0 +1,24 @@
+module Crud
+  require "bcrypt"
+
+  def Crud.create_hash_digest(password)
```

Comment:
```suggestion
  defcreate_hash_digest(password)
```
You don't need the `Crud` here. Please refacotr this file

Reply:
Replaced `Crud` with `self` here. This PR was using the `.self` to call the module methods before the following PR replaced it with the use of mixins.